test/kokoro/xds_k8s_lb.sh

```diff
@@ -168,6 +169,9 @@ main() {
     "remove_neg_test"
     "round_robin_test"
   )
+  if [[ "${TESTING_VERSION}" == "master" ]]; then
```

Comment:
Ah, that's on me. This would've worked when merged. The reason the test wasn't running is for manual test run we set TESTING_VERSION to `dev-master`, and not `master`.

Reply:
Done.